lumeer-core/src/main/java/io/lumeer/core/facade/UserNotificationFacade.java

```diff
@@ -126,6 +137,9 @@ public UserNotification updateNotification(final String notificationId, final Us
          data.append(UserNotification.CollectionShared.COLLECTION_NAME, resource.getName());
          data.append(UserNotification.CollectionShared.COLLECTION_ICON, resource.getIcon());
          data.append(UserNotification.CollectionShared.COLLECTION_COLOR, resource.getColor());
+
+         final String query = "{\"s\":[{\"c\":\"" + resource.getId() + "\"}]}";
```

Comment:
Maybe it will be better to use Query object and customize `toString()` method for this purpose. It's not good idea to hardcode important strings like this.